### ISSUE TITLE
can you share data ?

### ISSUE THREAD

**Issue by Sandy4321** [patrickjohncyh/gradient-recs#1]
7FashionCLIP weights and training code will be released with the original publication. At the moment of writing
this paper, the original training dataset is scheduled to be
released as well:

can you share data ?

**Comment by patrickjohncyh** [comment]
Thanks for your interest Sandy4321! We are still awaiting the official release of the dataset at this moment. Sit tight!

**Comment by Sandy4321** [comment]
You Closed #1 <https://github.com/patrickjohncyh/gradient-recs/issues/1> as
completed.
Does it means data already available
I can not find data in Git hub repo
